### ISSUE TITLE
Add Knowledge Portal gene list to L2G Goldstandards

### ISSUE THREAD

**Issue by xyg123** [opentargets/issues#4159]
I want to increase the size of the L2G training set because it will improve model performance and stability across releases.

## Background

The L2G training set has remained largely unchanged since its initial conception during the OpenTargets Genetics era, while small shuffles have been made regarding which studyLocusId are used in the training step, the input goldstandard trait-gene list has remained consistent. 

There is a large amount of curated gene lists publicly available on the [Knowledge portal resource](https://kp4cd.org/).  This ecompasses a range of phenotypic traits that match our GWAS resources and could be integrated with the L2G Goldstandards.  However, the gene lists are in a variety of different formats and will require a manual curation effort initially.  

This work also provides an opportunity to implement an automated studyLocusId selection process for the Goldstandards.

## Tasks

- [ ] Identify relevant gene lists from the Knowledge portal
- [ ] Curate a set of high-confidence goldstandards from the gene lists
- [ ] Identify most effective studyLocusIds to assign for the new goldstandards
- [ ] Potentially investigate an automatic procedure that selects the best powered studyLocusIds for the training set, which can improve model performance with the addition of new GWAS.

## Acceptance tests

How do we know the task is complete? 

There will be an increase in the goldstandard set size.

**Comment by addramir** [comment]
I believe this is not relevant anymore, since it is going to be covered by PEGASUS.
https://www.biorxiv.org/content/10.64898/2026.06.16.731894v1
